packages.yml

```diff
@@ -28,3 +28,12 @@ packages:
 
   - package: fivetran/amazon_ads
     version: [">=0.1.0", "<0.2.0"]
+
+  # - package: fivetran/reddit_ads
+  #   version: [">=0.1.0", "<0.2.0"]
```

Comment:
obligatory reminder to update before release.

Reply:
updated